saas/axops/src/applatix.io/axops/configuration/configuration.go

```diff
@@ -66,6 +108,16 @@ func CreateConfiguration(config *ConfigurationData) *axerror.AXError {
 }
 
 func UpdateConfiguration(config *ConfigurationData) *axerror.AXError {
+	//Check whether this is configured as Kubernetes secrets
```

Comment:
Another update scenario is when a config is updated from being a secret to just a config. In that case you would want to delete the kubernetes secret.